source/sass/global.scss

```diff
@@ -83,7 +83,7 @@ footer.site-footer {
     @media (min-width: $bp-lg) {
       column-count: 2;
       li {
-        break-inside: avoid-column;
+        break-inside: avoid;
```

Comment:
This was just the wrong value for the property, and as such wasn't doing anything. It fixes a small issue in the Mozfest Footer that's currently live.